portfolio/src/main/data_processing/processor.py

```diff
@@ -0,0 +1,77 @@
+""" Copyright 2019 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. 
+
+processor.py is used to clean csv's and create data suitable to be
+stored within a database."""
+
+from google.cloud import datastore
+import pandas as pd
+import numpy as np
+import os
+
+def cleanData(raw_file, COLUMNS_OF_INTEREST):
+  raw_file_path = 'raw_data/' + raw_file
+  dataframe = pd.read_csv(raw_file_path, index_col=0)
```

Comment:
Make sure this is leaving out the index and not the names. The index is not always numerical.

Reply:
I don't believe this messes up the indexing, and even if it does, I use a counter to iterate through the rows instead of the actual dataframe indexes.